CHANGELOG.md

```diff
@@ -6,6 +6,38 @@
 
 ### Fixes
 
+## v1.28.0 (2023-03-13)
+
+- Bump jsdom from 21.1.0 to 21.1.1 ([@dependabot](https://github.com/dependabot) in [#1257](https://github.com/jaegertracing/jaeger-ui/pull/1257))
```

Comment:
I think we should remove these dependabot changes.

Reply:
Removed, although the previous release lists some of them. 

We should document this in the release.md.

Reply:
Yeah, I suspect they might have been relevant to the patch release, though.